scripts/scan-o-matic

```diff
@@ -71,7 +71,7 @@ if __name__ == "__main__":
         default=False,
         action='store_true',
         help="Run in debug-mode. This makes the app vunerable and insecure" +
-        " and should be used behind firewall"  +
+        " and should be used behind firewall" +
```

Comment:
next line should probably have a leading space in the string

Reply:
I hade no intention to change the string at all just a automatic white space removal. String changed.